### DISCUSSION TITLE
ERROR: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.deno-ts(7026) (solved)

### DISCUSSION THREAD

**Discussion by alamo42** [freshframework/fresh#1345] [General]
I was about to create an issue regarding this error (not sure if this is the place for this info), but eventually found a solution. 

For anyone else having this issue when creating a new project, just create the project on some other directory, like desktop, then `rm -rf` the directory you were using on the first place and move the new directory to its location.

### Steps to reproduce:
1. Create a new fresh project as stated on the website: `deno run -A -r https://fresh.deno.dev .` (Accept to use **twind** and **VS Code** integration)
2. Run deno task start
3. Open `routes/index.tsx` on vs code

The index.tsx has errors related to JSX Types:
* Property 'children' is missing in type '{}' but required in type 'HeadProps'.deno-ts(2741)
* JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.deno-ts(7026)

### Notes
* Same as #1054, but it was marked as closed without proposing actual solutions.

### Tried Solutions
* Reinstalling/Reenabling Deno VSCode Extension
* Creating the project again

**Comment by newwingbird** [comment]
I have the same problem.
I tried the listed solutions on this issues https://github.com/denoland/fresh/issues/1477 but it dose not work.

**Comment by nobir** [comment]
I have found a solution that worked for me. Just set the `jsxImportSource` to `https://esm.sh/preact@10.19.6`. After that, restart your code editor.


Before
```json
"compilerOptions": {
  "jsx": "react-jsx",
  "jsxImportSource": "preact"
},
```


After (Solution)
```json
"compilerOptions": {
  "jsx": "react-jsx",
  "jsxImportSource": "https://esm.sh/preact@10.19.6"
},
```

## Software Information
**OS**: Windows 10 Pro
**Deno**: 1.42.3
**Fresh**: 1.6.8

**Reply by marvinhagemeister** [reply]
FYI: This was a recent regression in the LSP. Updating to canary like `deno upgrade --canary` or waiting for the next release should solve this.
